### PR TITLE
ImgFrame: report camera settings (focus, exposure, iso)

### PR DESCRIPTION
**Pull request by alex-luxonis**
Add new API which can be used on `ImgFrame` packets received from camera:
- getExposureTime()
- getSensitivity()
- getLensPosition() - ~~(note: lens position may be reported one frame in advance at the moment)~~ should be fixed
- getColorTemperature()

FW updates:
- fix black image when initial manual exposure was set with other controls **-- was merged previously**
- fix switching from auto to manual exposure causing the lens to jump to zero 
  (note: the lens is directly set now with manual focus, bypassing 3A) **-- was merged previously**
  
~~WIP, currently only reports values for ColorCamera.~~ - Fixed

Related PR: https://github.com/luxonis/depthai-shared/pull/51

### CODE REVIEW

File: src/pipeline/datatype/ImgFrame.cpp [hunk 1]
```diff
@@ -43,6 +43,15 @@ unsigned int ImgFrame::getHeight() const {
 RawImgFrame::Type ImgFrame::getType() const {
     return img.fb.type;
 }
+int ImgFrame::getExposureTime() const {
+    return img.cam.exposureTimeUs;
+}
```
Comment (themarpe) [inline src/pipeline/datatype/ImgFrame.cpp:48]: IMO duration would be more applicable in this case
```
std::chrono::microseconds ImgFrame::getExposureTime() const {
    return std::chrono::microseconds(img.cam.exposureTimeUs);
}
```

**Review by themarpe — CHANGES_REQUESTED** [review]

**Review by themarpe — CHANGES_REQUESTED** [review]
Seems like `video` frames don't get this information applied to it, but `preview` do. 
TODO - check how FW handles this.

**Comment by Erol444** [comment]
This (newer) PR also includes features added in this PR: https://github.com/luxonis/depthai-core/pull/210

**Comment by alex-luxonis** [comment]
> Seems like `video` frames don't get this information applied to it, but `preview` do.
`VideoEncoder` didn't forward the new ImgFrame metadata, was fixed.

**Pull request merged**